rtengine/imagesource.h

```diff
@@ -42,7 +42,7 @@ namespace rtengine
 
 class ColorTemp;
 class DCPProfile;
-class DCPProfileApplyState;
+class DCPProfile::ApplyState;
```

Comment:
You can't forward declare an inner class. That was the reason to separate it as `DCPProfileApplyState`. Now you've got to include `dcp.h` in `imagesource.h` which is included in ...